platform/features/layout/src/elements/LineProxy.js

```diff
@@ -21,8 +21,8 @@
  *****************************************************************************/
 
 define(
-    ['./ElementProxy', './LineHandle'],
-    function (ElementProxy, LineHandle) {
+    ['./ElementProxy', './LineHandle','./AccessorMutator'],
+    function (ElementProxy, LineHandle,AccessorMutator) {
```

Comment:
I think the spacing here will cause builds failures on code style checks. You can check with `gulp checkstyle` and `gulp lint`; `gulp verify` will run all verification checks including both of these. `gulp fixstyle` is a handy shortcut to fix checkstyle errors, but any lint errors will need to be handled manually.